### PR TITLE
rename estimator names

### PR DESCRIPTION
**Pull request by CrystalXuR**
rename estimator names for the base functions before

### CODE REVIEW

File: r-package/survlearners/R/surv_rl_grf.R [hunk 1]
```diff
@@ -47,38 +47,38 @@
 #' Y <- pmin(failure.time, censor.time)
 #' D <- as.integer(failure.time <= censor.time)
 #'
-#' rgrf_fit = rgrf(X, W, Y, D, times, p_hat = 0.5)
-#' rgrf_cate = predict(rgrf_fit, X)
+#' surv_rl_grf_fit = surv_rl_grf(X, W, Y, D, times, p_hat = 0.5)
+#' cate = predict(surv_rl_grf_fit)
```
Comment (erikcs) [inline r-package/survlearners/R/surv_rl_grf.R:51]: ideally add a 2nd line for each
cate.test = predict(surv_rl_grf_fit, X.test)
to make sure this too runs without problem before more tests are added

Comment (CrystalXuR) [reply r-package/survlearners/R/surv_rl_grf.R:51]: Good point, will do! thanks

File: r-package/survlearners/R/surv_fl_grf.R [hunk 2]
```diff
@@ -23,9 +23,11 @@
 #' censor.time <- (numeratorC/(4^2))^(1/2)
 #' Y <- pmin(failure.time, censor.time)
 #' D <- as.integer(failure.time <= censor.time)
+#' X.test <- matrix(rnorm(n * p), n, p)
```
Comment (erikcs) [inline r-package/survlearners/R/surv_fl_grf.R:26]: if you use n.test = something other than n then that is a nice simple extra test that everything works with test data different size than trainig data

Comment (CrystalXuR) [reply r-package/survlearners/R/surv_fl_grf.R]: yes, done!

**Pull request merged**